app/src/main/res/values/settings_keys.xml

```diff
@@ -283,7 +283,8 @@
 
     <string name="feed_update_threshold_key">feed_update_threshold_key</string>
     <string name="feed_update_threshold_default_value">300</string>
-    <string name="feed_show_played_items_key">feed_show_played_items</string>
+    <string name="feed_show_watched_items_key">feed_show_watched_items</string>
```

Comment:
Note that changing the key will reset the setting for users updating the app, unless you create a settings migration. I would rather just keep the previous (inconsistent) key. Renaming the string is perfectly fine instead.
```suggestion
    <string name="feed_show_watched_items_key">feed_show_played_items</string>
```